### DISCUSSION TITLE
What is `meta` in nuxt-bridge option

### DISCUSSION THREAD

**Discussion by ms-fadaei** [nuxt/nuxt#18274] [Questions] [answered]
I read some code of the `nuxt-bridge` package and noticed the [meta](https://github.com/nuxt/framework/blob/main/packages/bridge/types.d.ts#L17) option. In the [Nuxt 3 docs](https://v3.nuxtjs.org/docs/directory-structure/nuxt.config#meta), I also see the meta option that is available to Nuxt 3 only. So what is the purpose of the meta option in nuxt-bridge when we can't use it on Nuxt 2?

**Comment by pi0** [comment, marked as answer]
In nuxt 2, we use vue-meta while in nuxt3, we use `@vueuse/meta`. This option provides nuxt 3 identical composable and components for  using in nuxt 2 (https://v3.nuxtjs.org/docs/usage/meta-tags). config is not supported.
